What's New in v9.4

chri2547 opened the pull request:
This PR describes new features and changes available in Grafana v9.4.

chri2547 committed:
What's New in v9.4

chri2547 committed:
Merge remote-tracking branch 'origin/main' into docs/whats-new-in-9.4

chri2547 committed:
Merge remote-tracking branch 'origin/main' into docs/whats-new-in-9.4

chri2547 committed:
moves alert email templating to v9.4 release

linoman committed:
Add whats new for SAT (#61151)

* Add whats new for SAT

svennergr committed:
Logs: Add logdetails redesign to "whats new" (#61361)

* update whats new

* update text and add image

* change `logline` to `log line`

Co-authored-by: Ivana Huckova <30407135+ivanahuckova@users.noreply.github.com>

Co-authored-by: Ivana Huckova <30407135+ivanahuckova@users.noreply.github.com>

eleijonmarck committed:
add: skip org role sync

joeyorlando committed:
add note about oncall rbac to whats new 9.4 docs

matyax committed:
Loki Datasource: Add query validation to What's new (#62012)

Loki Datasource: add query validation to what's new

alexanderzobnin committed:
Add note about SAML auto login (#61926)

* Add note about SAML auto login

* Apply suggestions from code review

Co-authored-by: Christopher Moyer <35463610+chri2547@users.noreply.github.com>

* Apply suggestions from code review

Co-authored-by: Christopher Moyer <35463610+chri2547@users.noreply.github.com>

* run prettier

Co-authored-by: Christopher Moyer <35463610+chri2547@users.noreply.github.com>

matyax committed:
What's New: Add image for the Loki Query Validation feature (#62358)

* What's new 9.4: add image for Loki query validation

* What's new 9.4: update image path

ivanahuckova committed:
Docs: Add logs sample to whats new in 9.4 (#62354)

* Add logs sample to whats new in 9.4

* Update path to image

* Update docs/sources/whatsnew/whats-new-in-v9-4.md

Co-authored-by: Matias Chomicki <matyax@gmail.com>

---------

Co-authored-by: Matias Chomicki <matyax@gmail.com>

natellium committed:
Add datasource connection page to whatsnew

imatwawana commented on docs/sources/whatsnew/whats-new-in-v9-4.md:
```diff
@@ -0,0 +1,85 @@
+
+## New data source connection page in Dashboards and Explore
+
+When you start your journey to create a dashboard or explore your data, but you have no data source connected yet, you’ll be presented with a page that informs you of this and guides you to set up a first connection.
```
```suggestion
When you start your journey to create a dashboard or explore your data, but you don't have a data source connected yet, you’ll be shown a page that tells you this and guides you to set up a first connection.
```

imatwawana commented on docs/sources/whatsnew/whats-new-in-v9-4.md:
```diff
@@ -0,0 +1,85 @@
+
+When you start your journey to create a dashboard or explore your data, but you have no data source connected yet, you’ll be presented with a page that informs you of this and guides you to set up a first connection.
+
+Administrators can choose between selecting one of the most popular data sources or viewing the full list of data sources. Editors are guided to contact their administrator to configure data sources. In both cases, there's also an option to continue without setting up a data source and to use sample data instead.
```
```suggestion
Administrators can choose between selecting one of the most popular data sources or viewing the full list of them. Editors are guided to contact their administrator to configure data sources. In both cases, there's also an option to continue without setting up a data source and to use sample data instead.
```

eleijonmarck committed:
add: skip_org_role_sync setting

natellium committed:
Update datasource connection whats new 9.4

Co-authored-by: Isabel <76437239+imatwawana@users.noreply.github.com>

natellium committed:
Update datasource connection whats-new-in-v9-4.md

Co-authored-by: Isabel <76437239+imatwawana@users.noreply.github.com>

frossano-grafana committed:
Update whats-new-in-v9-4.md

Enterprise features add on

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/grafana/grafana?pullRequest=59614) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/grafana/grafana?pullRequest=59614) before we can accept your contribution.<br/>**9** out of **10** committers have signed the CLA.<br/><br/>:white_check_mark: linoman<br/>:white_check_mark: svennergr<br/>:white_check_mark: chri2547<br/>:white_check_mark: joeyorlando<br/>:white_check_mark: matyax<br/>:white_check_mark: alexanderzobnin<br/>:white_check_mark: eleijonmarck<br/>:white_check_mark: ivanahuckova<br/>:white_check_mark: natellium<br/>:x: frossano-grafana<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/grafana/grafana?pullRequest=59614) it.</sub>

zsofiakm committed:
Added command palette, navigation

zsofiakm committed:
Merge branch 'docs/whats-new-in-9.4' of https://github.com/grafana/grafana into docs/whats-new-in-9.4

zsofiakm committed:
Update whats-new-in-v9-4.md

frossano-grafana committed:
Update whats-new-in-v9-4.md

frossano-grafana committed:
Merge branch 'docs/whats-new-in-9.4' of https://github.com/grafana/grafana into docs/whats-new-in-9.4

frossano-grafana committed:
enterprise updates

natellium committed:
add panel header redesign content

natellium commented on docs/sources/whatsnew/whats-new-in-v9-4.md:
```diff
@@ -0,0 +1,142 @@
+
+{{< figure src="/media/docs/grafana/FormatReportShare9.4.png" max-width="750px" caption="Scale factor feature in Share functionality" >}}
+
+## Dashboard panel redesign
```
@imatwawana Made a couple of changes because in the end we added the head-less panel. Feel free to change it tho :)

brendamuir committed:
Adds alerting content to whats new

brendamuir committed:
Adds alerting content to whats new

mjseaman committed:
Fix typo

Nav navigation -> New Navigation

brendamuir committed:
adds image to pause alerts

brendamuir committed:
Merge branch 'docs/whats-new-in-9.4' of https://github.com/grafana/grafana into docs/whats-new-in-9.4

natellium commented on docs/sources/whatsnew/whats-new-in-v9-4.md:
```diff
@@ -0,0 +1,208 @@
+
+Grafana’s new panel is available only for React-based panels; no Angular-based panels will have the redesign (i.e., they’ll have the old Graph and Table visualizations).
+
+{{< video-embed src="/media/docs/grafana/screen-recording-panel-header-redesign-whats-new-9-4.mp4>" >}}
```
We've added a view as it feels the new panel header is hard to grasp without it, but I understand it might be too heavy. Let us know if we should remove it or find something else.

zsofiakm committed:
Updated new nav description

zsofiakm committed:
Updated cmd palette screenshot

chri2547 commented on docs/sources/whatsnew/whats-new-in-v9-4.md:
```diff
@@ -0,0 +1,208 @@
+
+Welcome to Grafana 9.4! Read on to learn about [add short list of what's included in this release]. For even more detail about all the changes in this release, refer to the [changelog](https://github.com/grafana/grafana/blob/master/CHANGELOG.md).
+
+## Feature
+
+[Generally available | Available in experimental/beta] in Grafana [Open Source, Enterprise, Cloud Free, Cloud Pro, Cloud Advanced]
+
+Description. Include an overview of the feature and problem it solves, and where to learn more (like a link to the docs).
+
+> **Note:** You must use relative references when linking to docs within the Grafana repo. Please do not use absolute URLs. For more information about relrefs, refer to [Links and references](/docs/writers-toolkit/writing-guide/references/).
```
```suggestion
```

chri2547 committed:
Update docs/sources/whatsnew/whats-new-in-v9-4.md

chri2547 commented on docs/sources/whatsnew/whats-new-in-v9-4.md:
```diff
@@ -0,0 +1,201 @@
+
+Welcome to Grafana 9.4! Read on to learn about [add short list of what's included in this release]. For even more detail about all the changes in this release, refer to the [changelog](https://github.com/grafana/grafana/blob/master/CHANGELOG.md).
+
+
```
```suggestion
```

chri2547 committed:
Update docs/sources/whatsnew/whats-new-in-v9-4.md

brendamuir committed:
adds alerting screenshots

brendamuir committed:
Merge branch 'docs/whats-new-in-9.4' of https://github.com/grafana/grafana into docs/whats-new-in-9.4

mjseaman committed:
reorder and begin adding availability

Re-ordered updates into groups, and added availability descriptions to the first few updates.

zsofiakm committed:
Updated command palette availability

imatwawana commented on docs/sources/whatsnew/whats-new-in-v9-4.md:
```diff
@@ -0,0 +1,234 @@
+
+## Dashboard panel redesign
+
+_Available to Preview using the `newPanelChromeUI` feature toggle in all editions of Grafana_
```
```suggestion
_Available to preview using the `newPanelChromeUI` feature toggle in all editions of Grafana_
```

imatwawana committed:
fix capitalization

chri2547 committed:
copy edit

mjseaman committed:
add before you upgrade section

mjseaman committed:
merge copy edits

mjseaman committed:
add docs relref links

chri2547 committed:
makes prettier

chri2547 committed:
resolves merge conflict

chri2547 committed:
added before you begin boilerplate

Commit:
Add Enterprise Datasource updates / highlights

nmarrs committed:
Add what's new section on updates to the canvas panel

nmarrs committed:
Update canvas demo vid src to correct location

nmarrs committed:
Canvas: add small shoutout for new server element in whats new

chri2547 committed:
copy edit final contributions

chri2547 committed:
Merge branch 'docs/whats-new-in-9.4' of github.com:grafana/grafana into docs/whats-new-in-9.4
